functional-wheel-launcher/lib/Utils/Logging-test.pm

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+=head1 Utils::Logging-test
+
+C<Utils::Logging_test> - Save logs directly on the worker for offline upload via ulogs 
```

Comment:
You might also want to rename the file too, just in case, to avoid confusing Perl?

```suggestion
=head1 Utils::Logging_est

C<Utils::Logging_test> - Save logs directly on the worker for offline upload via ulogs 
```

Reply:
okay, will update this, thx